### PR TITLE
feat: hash body signed fetch

### PR DESCRIPTION
**Pull request by gonpombo8**
## What does this PR change?

...

## How to test the changes?

1. Launch the explorer
2. ...

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

**Comment by m3taphysics** [comment]
We will need to optimize the memory allocations before this is merged. @gonpombo8 how critical is it?

**Comment by gonpombo8** [comment]
> We will need to optimize the memory allocations before this is merged. @gonpombo8 how critical is it?

i dont know anything about what you are talking about 😂 .
 Can you or someone of the team tackle this ?

**Comment by m3taphysics** [comment]
@gonpombo8 I know! That is why I was asking so I can prioritize it with the team :)

File: Explorer/Assets/Scripts/SceneRuntime/Apis/Modules/SignedFetch/SignedFetchWrap.cs [hunk 1]
```diff
@@ -63,13 +65,28 @@ public object SignedFetch(string url, string body, string headers, string method
             });
         }
 
+        public static String sha256_hash(String value) {
```
Comment (pravusjif) [inline Explorer/Assets/Scripts/SceneRuntime/Apis/Modules/SignedFetch/SignedFetchWrap.cs:68]: let's make this private Gon

Comment (gonpombo8) [reply Explorer/Assets/Scripts/SceneRuntime/Apis/Modules/SignedFetch/SignedFetchWrap.cs:68]: done

**Review by m3taphysics — APPROVED** [review]

**Pull request merged**